src/main/scala/no/ndla/learningpathapi/controller/LearningpathController.scala

```diff
@@ -359,16 +367,19 @@ trait LearningpathController {
 
     post("/", operation(addNewLearningpath)) {
       val newLearningPath = extract[NewLearningPath](request.body)
-      val createdLearningPath: Option[LearningPath] = optLong("copy-from") match {
-        case Some(id) => updateService.newFromExisting(id, newLearningPath, usernameFromHeader)
-        case None => Some(updateService.addLearningPath(newLearningPath, usernameFromHeader))
-      }
+      val learningPath = updateService.addLearningPath(newLearningPath, usernameFromHeader)
+      logger.info(s"CREATED LearningPath with ID =  ${learningPath.id}")
+      halt(status = 201, headers = Map("Location" -> learningPath.metaUrl), body = learningPath)
+    }
 
-      createdLearningPath match {
-        case None => halt(status = 404, body = Error(Error.NOT_FOUND, s"Learningpath with id ${params("copy-from")} not found"))
+    post("/copy/:path_id", operation(copyLearningpath)) {
```

Comment:
Jeg tenkte også på å extracte til to forskjellige case-classes, men følte at det kanskje blir litt rart at input-objektet forandrer seg avhengig av om et input-parameter (copy-from) er spesifisert eller ikke. Swagger dokumentasjonen blir vel da også inkonsistent, siden den vil si at Copyright er "required"